scipy/signal/ltisys.py

```diff
@@ -1744,12 +1744,15 @@ def freqresp(system, w=None, n=10000):
     >>> plt.plot(H.real, -H.imag, "r")
     >>> plt.show()
     """
-    if isinstance(system, TransferFunction):
+    if isinstance(system, (TransferFunction, ZerosPolesGain)):
         sys = system
     elif isinstance(system, lti):
         sys = system.to_tf()
```

Comment:
In principle, yes. However, right now, to get to this line the system must be state space, which means `to_zpk` is actually `system.to_tf().to_zpk()`, which doesn't win anything, so I didn't include it.

Though, if we put `to_zpk` here, then it's less likely we'll forget to change this when a proper `ss2zpk` is written, so I do agree it should change.